docs/en/quick-answers/troubleshooting/cant-use-file-linking-toolkit-app-chrome.md

```diff
@@ -11,10 +11,10 @@ lang: en
 
 Sometimes in Shotgun the action menu will not display the list of Toolkit applications available or will error when using local file linking, even with Shotgun Desktop running.  This doc is a guide to troubleshooting Shotgun’s connection to Desktop and will hopefully help you get things working.
```

Comment:
a couple general notes:
 - there are a few instances of *the* Shotgun Desktop in this doc. I'd clean those up. 
 - should we link to the Firefox doc here and to this doc in the FF doc? "This document explains blahlblah for the Chrome browser. If you're seeing issues with Firefox, see blah."

Reply:
Good idea I rewrote the introduction to become this: 
> This is a guide to troubleshooting Shotgun Desktop's browser integration in Chrome and will hopefully help you get things working. We have a separate guide for [Firefox](./cant-use-file-linking-toolkit-app-firefox.md).

And the exact same reverse sentence in the FF doc